frontend/src/components/Admin/AdminCandidateDecider.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState, useEffect } from 'react';
+import { Button, Form, Loader, Header, Message, Card, Checkbox } from 'semantic-ui-react';
+import csv from 'csvtojson';
+import styles from './AdminCandidateDecider.module.css';
+
+const mockInstances = [
+  {
+    name: 'Developer Spring 2022 Recruitment',
+    isOpen: true,
+    headers: [],
+    candidates: [],
+    uuid: 'asdfjkl'
+  },
+  {
+    name: 'Developer Fall 2022 Recruitment',
+    isOpen: true,
+    headers: [],
+    candidates: [],
+    uuid: 'hello-world'
+  }
+];
+
+const AdminCandidateDeciderBase: React.FC = () => (
+  <div id={styles.adminCandidateDeciderContainer}>
+    <CandidateDeciderInstanceCreator />
+    <CandidateDeciderInstanceList />
+  </div>
+);
+
+const CandidateDeciderInstanceCreator: React.FC = () => {
+  const [name, setName] = useState<string>('');
+  const [success, setSuccess] = useState<boolean>(false);
+  const [headers, setHeaders] = useState<string[]>([]);
+  const [responses, setResponses] = useState<any[]>([]);
+
+  const handleFileUpload = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (!e.target.files) return;
+    const file = e.target.files[0];
+    const reader = new FileReader();
+
+    reader.readAsText(file);
+    reader.onload = () => {
+      const str = reader.result as string;
+      const headers = str.slice(0, str.indexOf('\n')).split(',');
+      setHeaders(headers);
+      csv()
+        .fromString(str)
+        .then((parsedResponses) => setResponses(parsedResponses));
+    };
+  };
+
+  const handleSubmit = () => {
+    const instance = {
+      uuid: '',
+      name,
+      headers,
+      candidates: responses.map((res, i) => ({ id: i, responses: res, comments: [], ratings: [] })),
+      isOpen: true
+    };
+    setSuccess(true);
+    console.log(instance);
+    console.log('FORM SUBMITTED');
```

Comment:
There are few console.log() statements in this PR. Should these be removed/commented out before merging?

Reply:
These are placeholders for now since the API hasn't been implemented yet